tests/slsa_analyzer/ci_service/mock_repos/has_build_gh_actions/.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2022 - 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I noticed the `.yml` files were not checked for copyright headers. This PR [fixes](https://github.com/oracle-samples/macaron/pull/182/files) that.

Reply:
Sure, I will rebase and make sure to update this file's copyright header.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/136/commits/65d4d986e8bcc16c1a0a3ec3e112f01cd99ce878